client/state/selectors/get-site-comments.js

```diff
@@ -20,16 +20,17 @@ function filterCommentsByStatus( comments, status ) {
  * @param {Object} state Redux state
  * @param {Number} siteId Site for whose comments to find
  * @param {String} [status=unapproved] Status to filter comments
+ * @param {String} [order=asc] Order in which to sort filtered comments
  * @returns {Array<Object>} Available comments for site, filtered by status
  */
 export const getSiteComments = createSelector(
-	( state, siteId, status = 'unapproved' ) => {
+	( state, siteId, status = 'unapproved', order = 'asc' ) => {
```

Comment:
at this point we should be using an object for named parameters. it's going to be really confusing when we see…

```js
getSiteComments( state, siteId, undefined, 'desc' )
```

plus when we add the next option it becomes even more cooky.

```js
( state, siteId, { status = 'unapproved', order = 'asc' } ) => …
```